jenkinsfiles/Jenkinsfile.nightly

```diff
@@ -111,7 +111,7 @@ pipeline {
             steps {
                 dir("${TESTDIR}"){
                     sh './vagrant-ci-start.sh'
-                    sh 'ginkgo --focus="$(echo ${ghprbCommentBody} | sed -r "s/([^ ]* |^[^ ]*$)//" | sed "s/^$/Nightly/")" -v --failFast=${FAILFAST} -- -cilium.timeout=450m -cilium.provision=false -cilium.kubeconfig=${TESTDIR}/vagrant-kubeconfig -cilium.passCLIEnvironment=true -cilium.registry=$(./print-node-ip.sh)'
```

Comment:
How are we setting the registry if it's not passed on the commandline here?

Reply:
worth revisiting for some of the other jenkinsfiles as well, I think we need to inspect these 1-by-1 and make sure they're not breaking and accidentally pulling latest from docker.io.

Reply:
Answer: These are defaulting to `k8s1:5000` in the top of the `kubectl.go`.